app/contributing/style-guide.md

```diff
@@ -7,19 +7,21 @@ no_version: true
 
 |Best practice                      |Do                                             |Don't                                              |
 |---                                |---                                            |---                                                |  
+|Use US English (not British English)|The response **should** look like...          |The response **shall** look like...                |
```

Comment:
Might be worth linking to a dictionary for reference. The one I generally see linked for American spelling is https://www.merriam-webster.com/.